### DISCUSSION TITLE
Support conda packages

### DISCUSSION THREAD

**Discussion by kszucs** [community/community#25642] [Packages]
<p>Conda support custom channel hosted at custom channels <a href="https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/create-custom-channels.html" rel="nofollow noopener">https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/create-custom-channels.html</a></p>
<p>There are a couple of alternative hosting services which support conda repositories and it would be a nice addition since conda is gaining popularity.</p>

**Comment by clarkbw** [comment, marked as answer]
<p>We don’t have plans to support Conda right now but do have <code>pip</code> on our roadmap for later in the year.</p>

**Comment by curtisma** [comment]
<p>Pip support is good, but Conda can support any language or even multiple languages.  I would like to second adding conda support.</p>
